### ISSUE TITLE
Moving to another reference manager

### ISSUE THREAD

**Issue by HosseinMBeigi** [jimmejardine/qiqqa-open-source#418]
How I can move my libraries containing files and tags into another reference manages like Mendeley

**Comment by huachuman** [comment]
I'm interested in moving from Zotero to Qiqqa...

**Comment by ZazuMC** [comment]
Would not recommend it. There's next to no support and barely any dev on the product now. I grudgingly moved away from Qiqqa last year and have been very glad I did. I kept hoping someone would pick it up and do something decent with it, but I finally gave up hope.

**Comment by zhangxiaoxing** [comment]
Indeed, it's entirely possible to use multiple reference managers simultaneously. That said, the fact that all the code is readily available is a huge benefit. I do regret not having been more involved in the development, but credit must go to the developers for their hard work.
